tests/baselines/reference/importHelpersInIsolatedModules.js

```diff
@@ -69,14 +69,40 @@ var C = /** @class */ (function () {
     return C;
 }());
 //// [script.js]
-var tslib_1 = require("tslib");
+var __extends = (this && this.__extends) || (function () {

```

Comment:
This is what I meant by this change affecting `--importHelpers`.

Reply:
I think this is a correct change. The file is written as a script and checks as a script, so the output definitely can’t import the helpers, since that would make it lose its scriptiness.